### PR TITLE
Added a test for uniqueness of CCEs.

### PR DESCRIPTION
**Pull request by matejak**
The PR is supposed to fail CI as long as the duplicate CCEs exist in the project.

### CODE REVIEW

**Comment by ggbecker** [comment]
Solves #4485

**Comment by jan-cerny** [comment]
First we need to solve all CCE duplicates, and only after that this can be rebased and merged

**Comment by shawndwells** [comment]
> First we need to solve all CCE duplicates, and only after that this can be rebased and merged

CCEs deduplicated in https://github.com/ComplianceAsCode/content/pull/4586

**Comment by matejak** [comment]
Rebased, but there is still some stuff to do:

```
CCE 80994-7 is included in files: 
 - applications/openstack/keystone_container/container_keystone_disable_user_account_days_inactive/rule.yml
 - linux_os/guide/system/auditing/auditd_configure_rules/audit_login_events/audit_rules_login_events_tallylog/rule.yml

CCE 81037-4 is included in files: 
 - linux_os/guide/system/accounts/accounts-session/user_umask/accounts_umask_etc_csh_cshrc/rule.yml
 - linux_os/guide/system/bootloader-grub2/grub2_pti_argument/rule.yml

CCE 82053-0 is included in files: 
 - linux_os/guide/system/permissions/files/permissions_important_account_files/file_owner_etc_gshadow/rule.yml
 - linux_os/guide/system/permissions/files/permissions_important_account_files/file_permissions_etc_gshadow/rule.yml
 - linux_os/guide/system/software/disk_partitioning/partition_for_tmp/rule.yml

CCE 82068-8 is included in files: 
 - linux_os/guide/services/ssh/ssh_server/sshd_use_approved_macs/rule.yml
 - linux_os/guide/system/permissions/partitions/mount_option_var_tmp_nodev/rule.yml

CCE 82073-8 is included in files: 
 - linux_os/guide/system/auditing/auditd_configure_rules/audit_privileged_commands/audit_rules_privileged_commands_newgidmap/rule.yml
 - linux_os/guide/system/auditing/auditd_configure_rules/audit_privileged_commands/audit_rules_privileged_commands_usernetctl/rule.yml

CCE 82965-5 is included in files: 
 - linux_os/guide/system/software/system-tools/package_sudo_installed/rule.yml
 - linux_os/guide/system/software/system-tools/package_tar_installed/rule.yml

CCE 82966-3 is included in files: 
 - linux_os/guide/system/software/system-tools/package_sudo_installed/rule.yml
 - linux_os/guide/system/software/system-tools/package_tar_installed/rule.yml

CCE 82967-1 is included in files: 
 - linux_os/guide/system/software/system-tools/package_sudo_installed/rule.yml
 - linux_os/guide/system/software/system-tools/package_tar_installed/rule.yml

CCE 82976-2 is included in files: 
 - linux_os/guide/system/software/system-tools/package_openscap-scanner_installed/rule.yml
 - linux_os/guide/system/software/system-tools/package_policycoreutils_installed/rule.yml

CCE 82977-0 is included in files: 
 - linux_os/guide/system/software/system-tools/package_openscap-scanner_installed/rule.yml
 - linux_os/guide/system/software/system-tools/package_policycoreutils_installed/rule.yml

CCE 82978-8 is included in files: 
 - linux_os/guide/system/software/system-tools/package_openscap-scanner_installed/rule.yml
 - linux_os/guide/system/software/system-tools/package_policycoreutils_installed/rule.yml
```

**Comment by matejak** [comment]
I have deduplicated another batch of CCEs.

**Comment by yuumasato** [comment]
Just checking with latest merge of #4599 
@openscap-ci test this please

File: linux_os/guide/system/software/system-tools/package_openscap-scanner_installed/rule.yml [hunk 1]
```diff
@@ -15,9 +15,9 @@ rationale: |-
 severity: medium
 
 identifiers:
-    cce@rhel6: 82978-8
-    cce@rhel7: 82977-0
-    cce@rhel8: 82976-2
```
Comment (jan-cerny) [inline linux_os/guide/system/software/system-tools/package_openscap-scanner_installed/rule.yml]: Please change this also in `rhel8/profiles/ospp.profile`.

File: linux_os/guide/system/software/system-tools/package_sudo_installed/rule.yml [hunk 2]
```diff
@@ -16,9 +16,9 @@ rationale: |-
 severity: medium
 
 identifiers:
-    cce@rhel6: 82967-1
-    cce@rhel7: 82966-3
-    cce@rhel8: 82965-5
```
Comment (jan-cerny) [inline linux_os/guide/system/software/system-tools/package_sudo_installed/rule.yml]: Please change this also in `rhel8/profiles/ospp.profile`.

File: linux_os/guide/system/software/system-tools/package_openscap-scanner_installed/rule.yml (hunk 1 above)
Comment (matejak) [reply linux_os/guide/system/software/system-tools/package_openscap-scanner_installed/rule.yml]: OK

File: linux_os/guide/system/software/system-tools/package_sudo_installed/rule.yml (hunk 2 above)
Comment (matejak) [reply linux_os/guide/system/software/system-tools/package_sudo_installed/rule.yml]: OK

**Comment by jan-cerny** [comment]
Thank you.

**Pull request merged**